src/utils/formHelperFunctions.ts

```diff
@@ -495,6 +495,15 @@ function getNewStepUrl(stepKey: any) {
   return location.pathname + location.search + `#${stepKey}`;
 }
 
+function getPrevStepUrl(curStep: any, stepMap: Record<string, string>) {
+  let newStepKey = stepMap[curStep.key];
+  if (!newStepKey) {
+    const prevCondition = curStep.previous_conditions[0];
+    if (prevCondition) newStepKey = prevCondition.previous_step_key;
+  }
+  return newStepKey ? getNewStepUrl(newStepKey) : '';
+}
+
```

Comment:
There was a bit of behavior related to this that I wasn't expecting.  I set up a form with 3 steps, each with a back and next button.  The last step had an additional button with a NEXT action to the first step.  So basically you could navigate in circles, but the last step also had a submit to end the flow.  Then I tested it and without doing anything else, clicked the back button on the first step and it went to the last step.  I was thinking that this feature worked only with history and I was not expecting this (because I had not been to the last step yet).  Not sure if this boundary case behavior is intended or not.